api/processor_api.py

```diff
@@ -72,6 +73,7 @@ def create_fastapi_app(self):
         app.include_router(config_router, prefix="/config", tags=["Config"], dependencies=dependencies)
         app.include_router(cameras_router, prefix="/cameras", tags=["Cameras"], dependencies=dependencies)
         app.include_router(areas_router, prefix="/areas", tags=["Areas"], dependencies=dependencies)
+        app.include_router(area_all_router, prefix="/all_areas", tags=["Areas"], dependencies=dependencies)
```

Comment:
I think that it should be called `/area_all`, otherwise it looks like the endpoint returns all the areas.

Reply:
Yes, you are right!